config/routes.rb

```diff
@@ -1,6 +1,6 @@
 Rails.application.routes.draw do
-  # Define your application routes per the DSL in https://guides.rubyonrails.org/routing.html
-
-  # Defines the root path route ("/")
-  # root "articles#index"
-end
+    user_posts = get 'posts/:id', to: 'posts#show', as: :posts
+    user_post = get 'posts/:id', to: 'posts#show', as: :post
+    get 'users/:id', to: 'users#show', as: :users
+    get 'user/:id', to: 'users#show', as: :user
+  end
```

Comment:
- [ ] Please make sure that you fix this routing error :point_down: Currently, your user index page is not rending since you have no route for the user index page :blush: 
  
  ![image](https://github.com/eudondian/Blog-Haven/assets/92303891/da89d723-e083-4aa2-9d37-225da645fbb9)